core-library/src/main/java/org/silverpeas/core/io/media/image/option/AnchoringPosition.java

```diff
@@ -21,20 +21,19 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.silverpeas.core.io.media.video;
-
-import org.silverpeas.core.util.ServiceProvider;
-
-public class VideoThumbnailExtractorProvider {
-
-  /**
-   * Default private constructor
-   */
-  private VideoThumbnailExtractorProvider() {
-  }
-
-  public static VideoThumbnailExtractor getVideoThumbnailExtractor() {
-    return ServiceProvider.getService(VideoThumbnailExtractor.class);
-  }
+package org.silverpeas.core.io.media.image.option;
 
+/**
+ * @author Yohann Chastagnier
+ */
+public enum AnchoringPosition {
+  NorthWest,
+  North,
+  NorthEast,
+  West,
+  Center,
+  East,
+  SouthWest,
+  South,
+  SouthEast
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)